modules/volunteers/sitesupport/hourTable.js

```diff
@@ -110,6 +36,90 @@ export default {
     }
   },
   methods: {
+    load() {
+      this.records = null;
+      this.totalHours = 0;
+      this.totalSpentHours = 0;
+      this.hours = [];
+      this.totalVolunteers = null;
+
+      showSpinner();
+      this.summeryColumns = [
+        {value:'name', title:'Department Worked', source:null},
```

Comment:
Do we need to set `source` if it is not using anything? Would simply omitting it be an option?

Reply:
We could, and just change the check in the code later.  Feels like six of one or half dozen of another, unless you are passionate about it.